app/engine/engine.ts

```diff
@@ -65,17 +65,24 @@ export function getWinningBuzzer(buzzes: Map<string, number>, tiebreakerSeed?: s
     return undefined;
   }
 
-  const quantizedBuzzes: [string, number][] = validBuzzes.map(([userId, deltaMs]) => [
+  // generate 53-bit hash, discard MSBs to get 32-bit unsigned
+  const tiebreakSeed32 = cyrb53(tiebreakerSeed ?? "t") >>> 0;
```

Comment:
why does it need to be shifted?

Reply:
cyrb53 seeding requires 32-bit unsigned int

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Unsigned_right_shift
If the left operand is a number with more than 32 bits, it will get the most significant bits discarded.